README.md

```diff
@@ -3,148 +3,193 @@
 
 # Plankton
 
-Plankton is a Python library that assists in building simulated hardware devices. It is compatible with both Python 2 and 3.
-Currently this repository contains a simulated neutron chopper as it will be present at [ESS](http://europeanspallationsource.se).
-Choppers at ESS are abstracted in such a way that all of them are exposed via the same interface,
-regardless of manufacturer. The behavior of this abstraction layer can be modelled as a finite state machine.
+Plankton is a Python framework for simulating hardware devices. It is compatible with both Python 2 and 3.
 
-The docs-directory contains an `fsm`-file (created using the program [qfsm](http://qfsm.sourceforge.net/)) which describes
-the state machine. While this is still work in progress, it gives an idea of how the choppers are going to operate internally.
+Plankton can be run directly using Python 2.7 or 3.x, or using a prepackaged Docker image that includes all dependencies. See relevant usage sections for details.
 
-# Chopper simulation
+Resources:
+- [GitHub](https://github.com/DMSC-Instrument-Data/plankton)
+- [DockerHub](https://hub.docker.com/r/dmscid/plankton/)
+- [Dockerfile](https://github.com/DMSC-Instrument-Data/plankton/blob/master/Dockerfile)
 
-There are two ways of installing and running the chopper simulation. The first option is to run it directly using Python,
-the second is to run it in a [Docker containter](https://www.docker.com/).
 
-## Installation and startup I: Python
+## Purpose and Use Cases
 
-Clone the repository in a location of your choice:
+Plankton is being developed in the context of instrument control at the [ESS](http://europeanspallationsource.se), but it is general enough to be used in many other contexts that require detailed, stateful software simulations of hardware devices.
 
-```
-$ git clone https://github.com/DMSC-Instrument-Data/plankton.git
-```
+We consider a detailed device simulation to be one that can communicate using the same protocol as the real device, and that can very closely approximate real device behaviour in terms of what is seen through this protocol. This includes gradual processes, side-effects and error conditions.
+
+The purpose of Plankton is to provide a common framework to facilitate the development of such simulators. The framework provides a common set of tools and abstracts away protocol adapters, which helps minimize code replication and allows the developer of a simulated device to focus on capturing device behaviour.
+
+Potential use cases for detailed device simulators include:
+
+- Developing and testing software that interfaces with the device when it is unavailable
```

Comment:
Good idea, it's not really clear... that sentence is tough to word in a way that immediately makes sense.